daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -338,6 +338,34 @@ func addBlockHelper(t *testing.T) (v2.Handlers, echo.Context, *httptest.Response
 	return handler, c, rec, stx, releasefunc
 }
 
+func TestGetBlockTxids(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
+
+	handler, c, rec, stx, releasefunc := addBlockHelper(t)
+	defer releasefunc()
+
+	err := handler.GetBlockTxids(c, 0)
+	require.NoError(t, err)
+	require.Equal(t, 200, rec.Code)
+
+	c, rec = newReq(t)
+	err = handler.GetBlockTxids(c, 2)
+	require.NoError(t, err)
+	require.Equal(t, 404, rec.Code)
+
+	c, rec = newReq(t)
+	err = handler.GetBlockTxids(c, 1)
+	require.NoError(t, err)
+
+	var response model.BlockTxidsResponse
```

Comment:
Maybe add a sanity check against the genesis block as well?

Reply:
I called the endpoint on [block 0](https://github.com/algorand/go-algorand/pull/5611/files#diff-c9875030556c016ef35a4b47373f5005ee03b961a82157bed9ef734881edddc3R348) (I think this is the genesis block) above, but do let me know if you want to see some specific behavior checked.

Reply:
done 👍  [b38c1b9](https://github.com/algorand/go-algorand/pull/5611/commits/b38c1b99e1c45ab3eb267c0d917d0ab0010b1da5)